djangocms_blog/cms_plugins.py

```diff
@@ -17,15 +18,18 @@ class BlogPlugin(CMSPluginBase):
     module = get_setting('PLUGIN_MODULE_NAME')
 
     def get_render_template(self, context, instance, placeholder):
+
```

Comment:
I would prefer setting an empty list at first, appending the custom templates (if defined) and then appending the default ones